### ISSUE TITLE
Unable to get to hoobs.local from pc.

### ISSUE THREAD

**Issue by djhurt21** [hoobs-org/HOOBS#1241]
Setting up Hoobs for first time.  Followed instructions.  Cannot get to Hoobs.local

**Comment by timantoniadis** [comment]
Same here. I have been running hoobs for years now. However, I tried to login on my pc and Hoobs.local is not available. Any suggestions?

Chrome says;
This site can’t be reached
Check if there is a typo in hoobs.local.
DNS_PROBE_FINISHED_NXDOMAIN

Thanks a lot.

**Comment by timantoniadis** [comment]
I was having the same issue. Found this a bit further down which worked for me on a mac. 

Try accessing the Box via terminal and retrieve the logs

Here are the instructions to retrieve the logs:

Login to the HOOBS Box via SSH on your computer.
On MacOS use the Terminal app and type: ssh hoobs@hoobs.local

Type yes to accept the certificate key then press enter. When prompted for password, type hoobsadmin

On Windows, use Putty with the same credentials as above.

After successfully logging in, type this command: sudo hoobs service log
Copy and paste the output

**Comment by mkellsy** [comment]
If hoobs.local doesn't work try
http://hoobs

**Comment by JMYeager85** [comment]
Same thing here. 

This site can’t be reachedCheck if there is a typo in hoobs.local.
If spelling is correct, try running Windows Network Diagnostics.
DNS_PROBE_FINISHED_NXDOMAIN

I have tried SSH into the server to retrieve logs and get "destination is unreachable"

I have reformatted the SD card and am running it on a Raspberry Pi 4

I can see it on my network via my router, but I cannot ping or connect to it.
